### PR TITLE
	add global metadata and global metadata file to msbuild targets

### PR DESCRIPTION
**Pull request by herohua**
@vicancy @vwxyzh

### CODE REVIEW

File: src/nuspec/docfx.msbuild/build/docfx.msbuild.targets [hunk 1]
```diff
@@ -46,6 +46,8 @@
       <DocGenerateCommand>&quot;$(BuildDocToolPath)&quot; &quot;$(DocfxConfigFile)&quot; -o &quot;$(OutputFolder)&quot; -l &quot;$(LogFile)&quot; --logLevel &quot;$(LogLevel)&quot;</DocGenerateCommand>
       <DocGenerateCommand Condition="$(RebuildDoc)">$(DocGenerateCommand) -f</DocGenerateCommand>
       <DocGenerateCommand Condition="'$(DocTemplate)' != ''">$(DocGenerateCommand) --template &quot;$(DocTemplate)&quot; </DocGenerateCommand>
+      <DocGenerateCommand Condition="'$(GlobalMetadata)' != ''">$(DocGenerateCommand) --globalMetadata &quot;$(GlobalMetadata)&quot; </DocGenerateCommand>
```
Comment (vicancy) [inline src/nuspec/docfx.msbuild/build/docfx.msbuild.targets]: looks like we can provide a $(DocParameters) parameter instead

Comment (herohua) [inline src/nuspec/docfx.msbuild/build/docfx.msbuild.targets]: changed to general $(DocParameters)

**Comment by vicancy** [comment]
LGTM. please rebase and make only one commit to dev.

**Comment by herohua** [comment]
rebased and directly pushed to dev

**Pull request closed without merging**